### DISCUSSION TITLE
Dropping a response body to avoid data usage

### DISCUSSION THREAD

**Discussion by Ricardonacif** [nodejs/undici#812] [Q&A] [answered]
Hey, quick question.

After reading the headers, is there a way I can drop/ignore the response body (in order to avoid network data usage) without killing the entire socket? Is this part of HTTP protocol? What TCP package exactly should be sent?

**Comment by ronag** [comment, marked as answer]
> After reading the headers, is there a way I can drop/ignore the response body (in order to avoid network data usage) without killing the entire socket?

No, that is not possible with HTTP/1.1.

**Reply by Ricardonacif** [reply]
Thanks!
